src/lib.rs

```diff
@@ -824,5 +914,41 @@ macro_rules! create_app {
                             .required(true),
                     ),
             )
+            .subcommand(
+                Command::new("sign-eif")
+                    .about("Sign EIF with the given key")
+                    .arg(
+                        Arg::new("eif-path")
+                            .long("eif-path")
+                            .help("Path pointing to a prebuilt Eif image")
+                    )
+                    .arg(
+                        Arg::new("signing-certificate")
+                            .long("signing-certificate")
+                            .help("Local path to developer's X509 signing certificate."),
+                    )
+                    .arg(
+                        Arg::new("private-key")
+                            .long("private-key")
```

Comment:
If we decide to retain the local signing option, this should be --private-key-path for clarity.

Reply:
it will be inconsistent with `build-eif` experience and can bring more confusion then benefits